RevenueCatUI/CustomerCenter/CustomerInfo+CurrentEntitlement.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerInfo+CurrentEntitlement.swift
+//
+//  Created by Cesar de la Vega on 17/7/24.
+
+import Foundation
+import RevenueCat
+
+extension CustomerInfo {
+
+    func currentEntitlement() -> EntitlementInfo? {
```

Comment:
Maybe this can be named something descriptive like `earliestExpiringAppStoreEntitlement()`? And perhaps a line of docs that says something like: 
>Returns the earliest expiring iOS App Store entitlement. If this CustomerInfo contains multiple lifetime entitlements and no expiring entitlements, the returned entitlement is undefined.